psydac/api/ast/evaluation.py

```diff
@@ -905,7 +923,7 @@ def _initialize(self):
         # ...
 
         # ...
-        Nj             = ScalarTestFunction(space, name='Nj')
+        Nj             = element_of(space, name='Nj')
```

Comment:
should we keep the name Nj hard coded here? or check if there's an atom of the same name, if it is the case, then we create a random string?

Reply:
Good point, but I think we should address this in a separate merge request. Would you like to create an issue for this?

Reply:
you can do it and add a link to our discussion

Reply:
See issue #55